documentation/docs/20-commands/40-sv-migrate.md

```diff
@@ -2,7 +2,7 @@
 title: sv migrate
 ---
 
-`sv migrate` migrates Svelte(Kit) codebases. It delegates to the [`svelte-migrate`](https://github.com/sveltejs/kit/blob/main/packages/migrate) package.
+`sv migrate` migrates Svelte(Kit) codebases. It delegates to the [`svelte-migrate`](npmjs.com/package/svelte-migrate) package.
```

Comment:
```suggestion
`sv migrate` migrates Svelte(Kit) codebases. It delegates to the [`svelte-migrate`](https://www.npmjs.com/package/svelte-migrate) package.
```

But also: Would it be more helpful to link to the location of the package in this repo, rather than to npm?

Reply:
Thanks for catching this!

> But also: Would it be more helpful to link to the location of the package in this repo, rather than to npm?

I don't think so, which is why I changed it. Is there a reason you think that might be more helpful?

Reply:
Is it even worth to mention that in the docs? It's more of an internal thing for me at least. Also does not look like we have a similar sentence for `check` in the docs.

Reply:
There are lots of articles, blog posts, tweets, etc. mentioning the two, so I think it's helpful for users to know what the difference is between them and how they relate

Reply:
Hmm. But I guess all the docs have been updated to refer to sv, so maybe the more important place to put this is the svelte-migrate readme